content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
```

Comment:
Why this title? I probably would have translated "survoler les bugs" as "Getting an overview of [jEdit] bugs". Having read the article, I assume you mean "trackers" to refer generically to bugs, tasks, and support tickets, but without that context, "trackers" is confusing. I would probably just pick one (bugs, maybe, since that's what you did in French).